assets/style.css

```diff
@@ -309,12 +264,9 @@ header {
   background: transparent;
```

Comment:
I believe this line can be removed.

Reply:
that's not actually in there x_X, that's just a note from github